airflow/executors/kubernetes_executor.py

```diff
@@ -304,10 +317,13 @@ def _validate(self):
                 'must be set for authentication through user credentials; '
                 'or `git_ssh_key_secret_name` must be set for authentication '
                 'through ssh key, but not both')
+        # pylint: enable=too-many-boolean-expressions
 
 
 class KubernetesJobWatcher(multiprocessing.Process, LoggingMixin):
+    """Watches for Kubernetes jobs"""
     def __init__(self, namespace, watcher_queue, resource_version, worker_uuid, kube_config):
+        """Initializes the object"""
```

Comment:
```suggestion
```
It is not necessary. We should keep the documentation in the class description

Reply:
Done